lib/rails_twirp/testing/integration_test.rb

```diff
@@ -17,25 +23,33 @@ def before_rpc(&block)
       @before_rpc << block
     end
 
-    def rpc(service, rpc, request, headers: nil)
+    def rpc(svc, rpc, request, headers: nil)
       @request = request
-      service = app.twirp.routes.services[service].to_service
+      input_class = svc.rpcs[rpc][:input_class]
+      output_class = svc.rpcs[rpc][:output_class]
 
-      rack_env = {
-        "HTTP_HOST" => "localhost"
+      content_type = Twirp::Encoding::PROTO
```

Comment:
I would either set this as a constant on the class or as a helper method. It gives the same nice readability, but doesn't imply `content_type` could be changed later in the method